scripts/smoke/index.js

```diff
@@ -1,33 +1,93 @@
 /** @todo migrate these to use the independent docs repository at https://github.com/withastro/docs */
 
-import fs from 'fs';
-import { execa } from 'execa';
-import { fileURLToPath } from 'url';
-
 // NOTE: Only needed for Windows, due to a Turbo bug.
 // Once Turbo works on Windows, we can remove this script
 // and update our CI to run through Turbo.
```

Comment:
Remove this comment?

Reply:
Done in https://github.com/withastro/astro/pull/2557/commits/ac11ab0d0a412ec57731434879889d7d7549cd29